CONTRIBUTING.md

```diff
@@ -98,7 +98,7 @@ Here are some links to gnupg documentation that might be useful for those workin
 
 #### Travis releases
 
-When creating a commit inside `master` branch, `travis` on successful build will publish new `deb` and `rpm` packages to [`bintray`][bintray].
+It was once the case that creating a commit inside `master` branch, `travis` on successful build should publish new `deb` and `rpm` packages to [`bintray`][bintray]. 
```

Comment:
We should also note that now it creates a release on tag. At least it should do so.

Reply:
ok I'll add that soon

Reply:
It now says:

    After you commit a tag that matches the pattern '^v.*$' and travis succeeds, 
    Travis should publish new `deb` and `rpm` packages to [`bintray`][bintray].